packages/inventory-compliance/src/Fixtures/Fixtures.js

```diff
@@ -6,6 +6,7 @@ export const remediationsResponse = {
 export const system = {
     id: 'aa9c4497-5707-4233-9e9b-1fded5423ef3',
     name: '3.example.com',
+    supported: true,
     testResultProfiles: [{
```

Comment:
I guess we'll need to add majorOsVersion here after https://github.com/RedHatInsights/compliance-frontend/pull/1019

Reply:
Yeah, we should. Right now there are no tests for the `ComplianceRemediationsButton` component or any that invoke it to cause a failure. 

We could add tests now, but I'd rather fix the issues with it for now and revisit it later to refactor it also with more support from the backend to not have to construct much data here in the frontend.